.gitignore

```diff
@@ -2,3 +2,5 @@
 __pycache__
 .venv/
 notes.txt
+emailtest.py
+stock_data_polygon.py
```

Comment:
why were these not added? tests are useful to have in the repo

Reply:
emailtest.py I'm pretty much using as "scratch paper" now, the only code inside is stuff I was testing for the new API I had to implement lol. stock_data_polygon.py is what I spent most of a couple days ago doing but then realized after I built everything that the base/free level only include 1/3 of the three indices I want for my code, and coincidentally the one that works is the one I was testing with the whole time gg